esmf_regrid/tests/unit/experimental/unstructured_scheme/test_cube_to_GridInfo.py

```diff
@@ -0,0 +1,77 @@
+"""Unit tests for miscellaneous helper functions in `esmf_regrid.experimental.unstructured_scheme`."""
+
+from iris.coords import DimCoord
+from iris.cube import Cube
+import numpy as np
+import scipy.sparse
+
+from esmf_regrid.esmf_regridder import Regridder
+from esmf_regrid.experimental.unstructured_scheme import _cube_to_GridInfo
+
+
+def _grid_cube_global(n_lats, n_lons):
+    lat_span = np.linspace(-90, 90, n_lats * 2 + 1)
```

Comment:
I'm sure you've mentioned this before, but please could you just remind me here (not in the code) what the benefit of using linspace is for calculating the grid.

Reply:
doing an equivalent operation with `np.arange` has an issue where it can be sensitive to rounding errors.